test/e2e/node-selector.bats

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bats
+
+source test/e2e/helpers.sh
+
+setup() {
+	load 'bats/support/load'
+	load 'bats/assert/load'
+	load 'bats/file/load'
+}
+
+teardown() {
+	run kubectl delete builds.shipwright.io --all
+	run kubectl delete buildruns.shipwright.io --all
+}
+
+@test "shp build node selector single label" {
+    # generate random names for our build
+	build_name=$(random_name)
+
+    # create a Build with node selector 
+    run shp build create ${build_name} --source-url=https://github.com/shipwright-io/sample-go --output-image=my-fake-image --node="kubernetes.io/hostname=node-1"
```

Comment:
If flag is renamed to `node-selector`, then this would need to be updated (and ditto the others in the test).